pkg/controllers/disruption/suite_test.go

```diff
@@ -1676,7 +1676,7 @@ var _ = Describe("Candidate Filtering", func() {
 		Expect(err.Error()).To(Equal(fmt.Sprintf("nodepool %q can't be resolved for state node", nodePool.Name)))
 		Expect(recorder.DetectedEvent(fmt.Sprintf("Cannot disrupt Node: NodePool %q not found", nodePool.Name))).To(BeTrue())
 	})
-	It("should not consider candidates that do not have the karpenter.sh/capacity-type label", func() {
+	It("should consider candidates that do not have the karpenter.sh/capacity-type label", func() {
```

Comment:
I'm a little confused here. Aren't we making sure that we don't consider these nodes that are missing the labels listed in the changes of this file

Reply:
Maybe I could factor it a bit better in the future (but this would require a bit of an overhaul on the candidate struct type), but effectively we just check the labels on the candidate only for Consolidation when it matters. We wouldn't do it for drift or other eventual methods.